src/helpers/schema/validationRules.js

```diff
@@ -120,4 +125,43 @@ export default {
       });
       return errors;
     }),
+  bio: Joi.string()
+    .min(20)
+    .error((errors) => {
+      errors.forEach((err) => {
+        switch (err.type) {
+          case 'string.min':
+            err.message = 'bio must be at least 20 characters long';
+            break;
+          case 'string.base':
+            err.message = 'bio must be a string';
+            break;
+        }
+      });
+      return errors;
+    }),
+  username: Joi.string()
+    .alphanum()
+    .min(2)
+    .regex(/^[A-Za-z]+[A-Z-a-z-0-9]*$/)
+    .error((errors) => {
```

Comment:
The validation rule for `username` has already been defined in the `validationRules.js` file in the `helpers>schema` folder. It's better to reuse that one instead of duplicating it here.

Reply:
> The validation rule for `username` has already been defined in the `validationRules.js` file in the `helpers>schema` folder. It's better to reuse that one instead of duplicating it here.

the validation for username is a little bit different from the one that is being passed when someone is registering. here it's an option for a person to update his/her username where there it was required. that is why they are defined differently

Reply:
Ok. got it!